tests/Integration/FulfillmentRequestTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Tests\Integration;
+
+use GbWeiss\includes\OrderController;
+use GbWeiss\includes\SettingsRepository;
+use Mockery;
+use Mockery\MockInterface;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+
+class FulfillmentRequestTest extends \WP_UnitTestCase
+{
+    use MockeryPHPUnitIntegration;
+
+    public function test_it_does_update_the_woocommerce_order_status()
+    {
+        /** @var SettingsRepository|MockInterface */
+        $settingsRepository = Mockery::mock(SettingsRepository::class);
+        $settingsRepository->allows(['getFulfilledState' => 'wc-fulfilled']);
+
+        /** @var MockInterface|WC_Order */
+        $order = Mockery::mock("WC_Order");
+        $order->allows("set_status");
+        $order->allows("get_status");
+        $order->allows("save");
+
+        $controller = new OrderController($settingsRepository);
+        $controller->updateOrderStatus($order, 'wc-fulfilled');
+        $order->shouldHaveReceived('set_status', ['wc-fulfilled']);
+        $order->shouldHaveReceived('save');
+    }
```

Comment:
I would visually separate the assert part of the test into a seperate block by inserting a blank line before. Tests usually can be separated into three blocks. Arrange, Act and Assert. I like to separate these blocks because it makes them easier to read in my optinon.

Arrange: Create test data, create mocks, create the subject under test, ...

Act: Perform the action that should be tested (`updateOrderStatus` in this case)

Assert: Perform the assertions for the test.

This would result in the following structure for me:
```suggestion
    public function test_it_does_update_the_woocommerce_order_status()
    {
        /** @var SettingsRepository|MockInterface */
        $settingsRepository = Mockery::mock(SettingsRepository::class);
        $settingsRepository->allows(['getFulfilledState' => 'wc-fulfilled']);

        /** @var MockInterface|WC_Order */
        $order = Mockery::mock("WC_Order");
        $order->allows("set_status");
        $order->allows("get_status");
        $order->allows("save");

        $controller = new OrderController($settingsRepository);

        $controller->updateOrderStatus($order, 'wc-fulfilled');

        $order->shouldHaveReceived('set_status', ['wc-fulfilled']);
        $order->shouldHaveReceived('save');
    }
```
I would structure all tests like that.